dprint.json

```diff
@@ -3,11 +3,16 @@
   },
   "markdown": {
   },
+  "exec": {
+    "associations": ["**/*.{rb,gemspec,rake}", "Rakefile", "Gemfile", "Steepfile"],
+    "rubocop": "bundle exec rubocop --autocorrect --stdin {{file_path}} --stderr",
+    "rubocop.associations": "**/**"
```

Comment:
Because of no supported multiple globs yet https://github.com/dprint/dprint-plugin-exec/blob/94ed0fe3fac6a631c5e3e00538f29c30770a0738/src/configuration.rs#L165-L173